_posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "DSAA Wrapped"
+description: |
+  Reflecting on 2022.
+author:
+  - name: Chloe Pou-Prom, Meggie Debnath, Derek Beaton
+date: 2022-12-13
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+The end of the year is a time for reflection, reminiscing, and sharing screenshots of our Spotify Wrapped. For those who are unfamiliar, Spotify Wrapped is a yearly marketing campaign by Spotify, where they compile streaming data and create a personalized graphic for each user. ^[We surveyed the members on DSAA and found that the most listened music genre is: ???] 
+
+In the spirit of data, end-of-year reflections, and colorful infographics, we thought it would be fun to create our very own DSAA wrapped!
+
+![](dsaa_wrapped.PNG)
+
+Enjoy some stats!!
+
+# Projects
+
+2022 was a busy year with lots of projects!!
+
+- Total \# of projects deployed: 7
```

Comment:
Is this number correct? I feel like there are more...
ED wait times, SMH IRT staffing optimization, chartwatch surgical, ops centre unit level tiles, hemodialysis, ICH

Reply:
ID consult, CHARTWatch 2.0.

Reply:
so maybe let's say at least 8-9 within the year. I do think there were some data fulfillment ones, too (e.g., a PACS one).

Reply:
We have 9 big ones **from July through December**: hemo, ID consult, CW2.0, Unit Level Tiles, SMH IRT, CW Surg, Ops Centre alerting, ED wait times, ICH